src/ImageSharp/IO/IFileSystem.cs

```diff
@@ -9,16 +9,32 @@ namespace SixLabors.ImageSharp.IO;
 internal interface IFileSystem
 {
     /// <summary>
-    /// Returns a readable stream as defined by the path.
+    /// Opens a file as defined by the path and returns it as a readable stream.
     /// </summary>
     /// <param name="path">Path to the file to open.</param>
-    /// <returns>A stream representing the file to open.</returns>
+    /// <returns>A stream representing the opened file.</returns>
```

Comment:
I changed this from "file to open" to "opened file" because the wording was misleading - it implied that the file isn't yet open when the method returns, when in fact it is.

Reply:
Yeah that's better.